update/common/scripts/5.4/cleanuntranslatablerelations.php

```diff
@@ -53,12 +53,17 @@
  */
 function parseRelationListIds( $relationListXml )
 {
+    if ( empty( trim( $relationListXml ) ) )
+        return array();
+
     $xml = simplexml_load_string( $relationListXml );
     $list = $xml->{'relation-list'};
 
     $contentIds = array();
-    foreach ( $list->children() as $item ) {
-        $contentIds[] = (int)$item['contentobject-id'];
+    if ( $list ) {
```

Comment:
```
if ( !$list ) {
    return array();
}
```

Also, try to use a strict test, against the "empty" value we might get from `$xml->{'relation-list'}`.